### ISSUE TITLE
Class not Found Exception

### ISSUE THREAD

**Issue by hannespreishuber** [Microsoft/project-rome#40]
Unhandled Exception:

Java.Lang.ClassNotFoundException: Didn't find class "md56b4bf6e49f378dfe4b7822922cda6c0f.RemoteSystemDiscoveryListener" on path: DexPathList[[zip file "/data/app/com.project_rome.romecastremote-1/base.apk"],nativeLibraryDirectories=[/data/app/com.project_rome.romecastremote-1/lib/arm, /system/fake-libs, /data/app/com.project_rome.romecastremote-1/base.apk!/lib/armeabi-v7a, /system/lib, /vendor/lib, /system/vendor/lib]]

**Comment by cmknox** [comment]
@hannespreishuber sorry for the delay.  which version are you using?  0.8.0?

**Comment by hannespreishuber** [comment]
yes, 0.8.0 but stopped working on that, so no idea if it is still a problem

**Comment by cmknox** [comment]
@hannespreishuber checking in to see what the latest is on this?
